Script/AtomicEditor/ui/Shortcuts.ts

```diff
@@ -99,10 +99,13 @@ class Shortcuts extends Atomic.ScriptObject {
 
     }
 
-    invokeGizmoAxisModeChanged(mode: Editor.AxisMode, toggle: boolean = false) {
-
-        this.sendEvent("GizmoAxisModeChanged", { mode: mode, toggle: toggle });
+    toggleGizmoAxisMode() {
+        var editor = EditorUI.getMainFrame().resourceframe.currentResourceEditor;
```

Comment:
exactly, we have that https://github.com/AtomicGameEngine/AtomicGameEngine/blob/master/Script/AtomicEditor/ui/EditorUI.ts#L51 :sweat_smile: